Dockerfile

```diff
@@ -1,8 +1,12 @@
-FROM ubuntu:16.04
+FROM ubuntu:18.04
 WORKDIR /app
 ENV TZ America/Sao_Paulo
 ENV LANG pt_BR.UTF-8
 RUN \
+  apt-get clean && \
+  apt-get update && \
+  apt-get install -y locales tzdata && \
```

Comment:
@merencia só por curiosidade mesmo? Porque precisou instalar estes pacotes no 18.04?

Reply:
Então, posso ter feito algo errado, mas quando fiz o build quebrou aqui:
https://github.com/ContaAzul/docker-ubuntu-br/pull/1/files#diff-3254677a7917c6c01f55212f86c57fbfL8
e aqui: 
https://github.com/ContaAzul/docker-ubuntu-br/pull/1/files#diff-3254677a7917c6c01f55212f86c57fbfL9

Instalei os pacotes para funcionar... 
Na pratica rolava o seguinte:
`/bin/sh: 1: locale-gen: not found` e `/usr/sbin/dpkg-reconfigure: tzdata is not installed`